org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/launch/GradleRunConfigurationAttributes.java

```diff
@@ -258,88 +290,83 @@ public static void applyShowConsoleView(boolean showConsoleView, ILaunchConfigur
         launchConfiguration.setAttribute(SHOW_CONSOLE_VIEW, showConsoleView);
     }
 
+    public static void applyWorkspaceOverride(boolean overrideWorkspaceSettings, boolean offlineMode, boolean buildScansEnabled, ILaunchConfigurationWorkingCopy launchConfiguration) {
```

Comment:
Sounds good. I checked the launch config UI's api and looks like we can do the simplification.  Here's the card for that: https://github.com/eclipse/buildship/issues/417